infra/terraform/dev/init/.gitignore

```diff
@@ -0,0 +1,32 @@
+# Local .terraform directories
```

Comment:
This looks similar to the other terraform gitignore file. Any chance / is it reasonable to merge them, and have these files reference the same with symlinks? Or just move it to a higher directory.